### PR TITLE
Add GedcomX Names and Rectangles for Them

### PR DESCRIPTION
**Pull request by zvakanaka**

### CODE REVIEW

File: src/components/sharedViewer/SharedViewer.js [hunk 1]
```diff
@@ -15,23 +16,71 @@ const sidePanelCSS = css`
...
+          .then((aceData) => {
+            setPersons(getPersons(aceData))
+            setLoading(false)
+          })
+          .catch(handleError)
+      })
+      .catch(handleError)
+  }, [arkPath, handleError])
+
+  useEffect(() => {
```
Comment (brinbh) [inline src/components/sharedViewer/SharedViewer.js:50]: should this useEffect be moved to the IndexImage component?

File: src/services/aceOrchestrationService.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { transcribeImage } from './cogsworthService'
+import { nlpMultirecord, segmentRecord } from './dexterService'
+import { downloadImage } from './dasService'
+
+async function getAceData(apid) {
+  const image = await downloadImage(apid)
+  const cogsworthData = await transcribeImage(image, apid)
+  const recordSegments = await segmentRecord(cogsworthData, apid)
+  const nlpData = await nlpMultirecord(recordSegments, apid)
+  return { cogsworthData, nlpData }
```
Comment (brinbh) [inline src/services/aceOrchestrationService.js:10]: I don't think we need to return the cogsworth data anymore here. And eventually, we will not need to return the nlp data, but the sls data. But I will add that.

File: src/services/dexterService.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import axios from '@fs/zion-axios'
+import Cache from '@fs/zion-cache'
+
+const dexterExperiments = []
+
+const storeName = 'dexter'
+const cache = new Cache({ storeName })
+
+/**
+ * Get records and their regions from Cogsworth data
```
Comment (brinbh) [inline src/services/dexterService.js:10]: Is this comment accurate? what does this function do exactly?

File: src/services/aceOrchestrationService.js [hunk 4]
```diff
@@ -1,15 +1,40 @@
...
+  const apid = await arkToApid(arkPath)
   const image = await downloadImage(apid)
   const cogsworthData = await transcribeImage(image, apid)
   const recordSegments = await segmentRecord(cogsworthData, apid)
   const nlpData = await nlpMultirecord(recordSegments, apid)
   return nlpData
 }
 
+function useNlpPersons(arkPath) {
+  const [persons, setPersons] = useState([])
```
Comment (brinbh) [inline src/services/aceOrchestrationService.js:16]: it seems a little odd to me setting the state in a function that is in another file. Is there a standard way to do this?

File: src/services/dasService.js [hunk 5]
```diff
@@ -19,7 +19,7 @@ const deepZoomCloudBase = `https://${sgBase}.familysearch.org/service/records/st
 async function arkToApid(ark) {
   try {
     const response = await axios.get(
-      `https://${sgBase}.familysearch.org/service/records/storage/dascloud/das/v2/3:1:${ark}/name?namespace=apid`
+      `https://${sgBase}.familysearch.org/service/records/storage/dascloud/das/v2/${ark}/name?namespace=apid`
```
Comment (patrickliechty) [inline src/services/dasService.js:22]: Why are we using sg urls when we can get the image from a familysearch.org url?

File: src/services/dasService.js [hunk 6]
```diff
@@ -51,9 +51,9 @@ async function downloadImage(apid) {
  * @returns {String} the image url
  */
 function getImageArkUrl(arkPath) {
-  let dzPath = 'www.familysearch.org/dz/v1/3:1:'
-  if (sgBase === 'sg31b0') dzPath = 'beta.familysearch.org/search/dasbridge/dz/v1/3:1:'
-  if (sgBase === 'sg32i0') dzPath = 'integration.familysearch.org/search/dasbridge/dz/v1/3:1:'
+  let dzPath = 'www.familysearch.org/dz/v1/'
+  if (sgBase === 'sg31b0') dzPath = 'beta.familysearch.org/search/dasbridge/dz/v1/'
```
Comment (patrickliechty) [inline src/services/dasService.js:55]: Same here.  This code could go away if we don't use sg urls.

Comment (zvakanaka) [reply src/services/dasService.js]: Thanks, I got some help from DAS and they gave me new URLs. We are now free of SG URLs.

**Pull request merged**